src/server/scripts/createRoom.js

```diff
@@ -1,4 +1,5 @@
 /* global RoomMethods:true math:true ProgramUtils:true RandomGenerate: true*/
+// TODO why is this commented out and why are all these globals here ^^^?
```

Comment:
Because, they used to be needed here, the logic was moved and the globals are needed anymore, I have no idea why use strict is commented out here, uncomment "use strict"; and remove all but RandomGenerate from the globals